gensim/models/fasttext.py

```diff
@@ -1338,63 +1144,21 @@ def get_vector(self, word, norm=False):
             else:
                 return word_vec
 
-    def init_ngrams_weights(self, seed):
-        """Initialize the vocabulary and ngrams weights prior to training.
+    def resize_vectors(self, seed=0):
+        """Make underlying vectors match 'index_to_key' size; random-initialize any new rows.
 
-        Creates the weight matrices and initializes them with uniform random values.
+        Unlike in superclass, the 'vectors_vocab' array is of primary importance, with
```

Comment:
I'm not sure if this comment is docstring-worthy. What important information does it convey to the reader? What decision can they now make based on that information?

Reply:
Ultimately, because users can and may access the `.vectors`/`.vectors_vocab` directly, this is worthwhile to note this dichotomy somewhere in the doc-comments – but yes, the API mention should be at a different level, with other notes in source. 

(This dichotomy, while necessary, is also a bit of a pain to navigate in the mmap-support, as in KV class it's definitely `.vectors` that must be live disk-backed, while in the FTKV class the purely-derived `.vectors` is not even saved separately – so an mmap-load wouldn't necessarily have a file to use. But still, the typical use-case for mmapping – many processes using the same model – would still want to share a `.vectors`.)